server/src/test/java/io/unitycatalog/server/utils/PopulateTestDatabase.java

```diff
@@ -0,0 +1,334 @@
+package io.unitycatalog.server.utils;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.unitycatalog.server.model.*;
+import io.unitycatalog.server.persist.CatalogRepository;
+import io.unitycatalog.server.persist.FunctionRepository;
+import io.unitycatalog.server.persist.HibernateUtil;
+import io.unitycatalog.server.persist.SchemaRepository;
+import io.unitycatalog.server.persist.dao.*;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+
+import java.util.*;
+
+import static io.unitycatalog.server.utils.ColumnUtils.*;
+
+public class PopulateTestDatabase {
```

Comment:
there needs to be instructions one how to run this class

Reply:
added

Reply:
how do you run the main method of this class?? Please be more detailed.

Reply:
Added details for the same.